.github/workflows/dynamic-ci.yml

```diff
@@ -47,15 +47,14 @@ jobs:
               - ".github/workflows/windows-macos.yml"
               - "3rd-party/**"
               - "completions/**"
-              - "data/**"
+              - "data/*"
```

Comment:
What does this change do in practice?

Reply:
We need to make sure CLI tests still trigger on this sort of PR (or just run on a schedule), once they are merged.